### ISSUE TITLE
Output a fingerprinting list

### ISSUE THREAD

**Issue by garvankeeley** [mozilla-services/shavar-list-creation#110]
The fingerprinting list is an intersection of the shavar fingerprinting list and some other list (I am not sure which one).
For iOS (and possibly other consumers), the fingerprinting list is needed as a dedicated list. Ideally this would be a simple JSON array of domains to simplify consumer parsing.

**Comment by svensevenslow** [comment]
Hi! I'd like to work on this issue. Could you please explain in a little bit more detail about what needs to be done?

**Comment by svensevenslow** [comment]
@skim1102  Hi! I'd like to work on this issue. Could you please explain in a little bit more detail about what needs to be done?

**Comment by garvankeeley** [comment]
Closing as this has been done, see https://github.com/mozilla-services/shavar-prod-lists/blob/72.0/normalized-lists/base-fingerprinting-track.json

**Comment by skim1102** [comment]
@svensevenslow, yes you can work on this one. As @garvankeeley pointed out, we need to generate the `base-fingerprinting-track.json` and create a PR on the shavar-prod-lists when changes happen on any of the following categories: `Advertising`, `Analytics`, `Social`, `Content`, or `Fingerprinting` as specified in the configuration for generating the `base-fingerprinting-track-digest256` [here](https://github.com/mozilla-services/shavar-list-creation-config/blob/master/prod.ini#L179). No change is needed on the diconnect_mappaing.json and the .ini files but you may want to generate the `base-fingerprinting-track.json` file when the `base-fingerprinting-track-digest256` is generated and use the GitHub API to create a PR to shavar-prod-lists when the files are checked and need to be published in `publish2cloud.py`.

To create a PR, I believe a Mozilla GitHub account credential is needed. We can get this information when the PR is ready to be merged. For now, please test using your cloned copy of shavar-list-creation and shavar-prod-lists.

**Comment by skim1102** [comment]
@svensevenslow, please note that the branches on shavar-prod-lists are versioned. Meaning `base-fingerprinting-track.json` should be generated using values in the respective versioned branch in shavar-prod-lists repo AND the JSON file should be uploaded to the respective versioned branches.

**Comment by svensevenslow** [comment]
Thank you for the detailed explanation, I'll get back to you if I have any queries while working on it.

**Comment by svensevenslow** [comment]
Hi @skim1102 ! I've opened a [ PR ](https://github.com/mozilla-services/shavar-list-creation/pull/128)for this issue, could you please have a look.
